viewing_party/party.py

```diff
@@ -0,0 +1,154 @@
+from collections import Counter
+
+
+def create_movie(title, genre, rating):
+    if title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
+
+    movie = {}
+    movie["title"] = title
+    movie["genre"] = genre
+    movie["rating"] = rating
+
+    return movie
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if title == movie["title"]:
+            user_data["watchlist"].remove(movie)
+            add_to_watched(user_data, movie)
+    return user_data
+
+
+def get_watched_avg_rating(user_data):
+    ratings_list = []
+
+    if len(user_data["watched"]) == 0:
+        return 0.0
+
+    for movie in user_data["watched"]:
+        ratings_list.append(movie["rating"])
+
+    average_rating = sum(ratings_list) / len(ratings_list)
```

Comment:
Nice way to use the sum function by extracting the ratings. You could also try getting the ratings as a list comprehension, or consider accumulating the sum yourself (to avoid having to allocate another list entirely).